test/e2e/framework/metrics/metrics_grabber.go

```diff
@@ -26,13 +26,23 @@ import (
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/util/wait"
 	clientset "k8s.io/client-go/kubernetes"
-	"k8s.io/kubernetes/pkg/master/ports"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	"k8s.io/kubernetes/test/e2e/system"
 
 	"k8s.io/klog"
 )
 
+const (
+	// insecureSchedulerPort is the default port for the scheduler status server.
+	// May be overridden by a flag at startup.
+	// Deprecated: use the secure KubeSchedulerPort instead.
+	insecureSchedulerPort = 10251
```

Comment:
can't use constants in `test/e2e/framework/ports.go` due to cyclical imports to `test/e2e/framework` :(

Reply:
ack, i was hoping that one day the component ports and similar constants can be exported to a repository somewhere.